pkg/sql/tree/transformer.go

```diff
@@ -1934,7 +1935,9 @@ func transformLoadDataStmtToLoad(lds *ast.LoadDataStmt) *Load {
 			ues[i] = transformAssignmentToUpdateExpr(a)
 		}
 	}
-
+	if !filepath.IsAbs(lds.Path) {
```

Comment:
It is suitable to move the ABS check into the function handleLoadData. In the future, the transformers will be removed after the new parser accomplished.